keyboards/handwired/slash/config.h

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright 2019 4sdftemp
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID 0xFEED
+#define PRODUCT_ID 0x0000
+#define DEVICE_VER 0x0001
+#define MANUFACTURER asdftemp
+#define PRODUCT BLESS
```

Comment:
Shouldn't this be "Slash", as that is what this board is referenced as everywherel else? 

```suggestion
#define PRODUCT Slash
```

Reply:
Good catch, Should be fixed now